### PR TITLE
Extract HTML from view_operations.php to a Twig template

### PR DESCRIPTION
**Pull request by thicolares**
### Description
- Extract HTML from view_operations.php to a Twig template
- Performed some local smoke tests
    - Rename a view
    - Delete a view

Related to #14801 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by thicolares** [comment]
The Travis CI build failed. I going to check this out in a few minutes

**Review by williamdes — APPROVED** [review]
LGTM

Thank you @thicolares

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Hi, @thicolares. Thanks for your contribution.

Could you rename the template to `templates/table/operations/view.twig`?

File: templates/view_operations.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <fieldset class="tblFooters">
+            <input type="hidden" name="submitoptions" value="1">
+            <input class="btn btn-primary" type="submit" value="{% trans 'Go' %}">
+        </fieldset>
+    </form>
+</div>
+
+<div>
+    <fieldset class="caution">
+        <legend>{% trans 'Delete data or table' %} novo</legend>
```
[the remark this answers is not in the document]
Comment (thicolares) [reply templates/view_operations.twig]: OMG. Thank you for that

File: view_operations.php [hunk 2]
```diff
@@ -142,17 +114,14 @@
...
-    'DROP VIEW',
-    __('Delete the view (DROP)'),
-    'drop_view_anchor'
-);
-echo '</ul>';
-echo '</fieldset>';
-echo '</div>';
+echo $template->render('view_operations', [
+    'hidden_inputs' => Url::getHiddenInputs($GLOBALS['db'], $GLOBALS['table']),
+    'table_name_in_html_entities' => htmlspecialchars($GLOBALS['table']),
```
[the remark this answers is not in the document]
Comment (thicolares) [reply view_operations.php]: Ok. I didn't know. Thank you for all your feedbacks

[the remark this answers is not in the document]
Comment (thicolares) [reply view_operations.php]: Ok. Thank you for all your feedbacks

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**